terraform/testdata/plan-ignore-changes/main.tf

```diff
@@ -1,9 +1,9 @@
 variable "foo" {}
 
 resource "aws_instance" "foo" {
-  ami = "${var.foo}"
+  ami = var.foo
```

Comment:
This was updated to remove warnings when I was in the neighborhood while fixing apply tests